keyboards/skyloong/gk61/pro_48/mcuconf.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2023 linlin012 (@linlin012)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef STM32_I2C_USE_I2C1
+#define STM32_I2C_USE_I2C1 TRUE
+
+#undef STM32_I2C_USE_DMA
+#define STM32_I2C_USE_DMA FALSE
```

Comment:
No changes needed, but why no DMA?  Wouldn't that adversely effect performance?

Reply:
Thank you for your prompt, I will modify it.